### DISCUSSION TITLE
Nomad exec?

### DISCUSSION THREAD

**Discussion by koder29406** [Kamilcuk/nomad-tools#6] [Q&A]
Hey!
This looks pretty nice!
Do you have any plans to implement `nomad exec` tools in this?

**Comment by Kamilcuk** [comment]
Hi.

I do not understand. What is "nomad exec tool"? Kindly explain in details. `nomad alloc exec` already exists.

I created `nomadtools go`, maybe it is what you expect. If that is, you could consider sharing if you think what is a better name for it and that name shouldn't clash with nonad cli subcommands, which is why it is not `run` nor `exec`.
